src/components/auth/AuthenticationOutage.stories.js

```diff
@@ -0,0 +1,13 @@
+import AuthenticationOutage from './AuthenticationOutage';
+
+export default {
+  title: 'Private API / Authentication Outage',
+  render: ({label}) => <AuthenticationOutage loginOption={{label}} />,
+};
```

Comment:
```suggestion
export default {
  title: 'Private API / Authentication Outage',
  component: AuthenticationOutage,
  args: {
    loginOption: {
      label: 'DigiD,
    },
  },
};
```
why not this?

The default storybook render is to pass the `args` into the `component` as props.